recipes/tpi/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "tpi" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e7dc5262d81f82dfe3def1f791769ed6024c0791ee52edc841906796dd01778
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - tpi = tpi.main:main
+
+requirements:
+  host:
+    - python >=3.4
+    - pip
+    - setuptools_scm >=3.4
+    - toml
+  run:
+    - python >=3.4
+    - sshfs >=2021.7.1
+    - funcy
+    - jinja2 >=2.0.0
+    - miutil >=0.7.0
+    - requests
+    - python-terraform >=0.10.1
+    - setuptools
+
+test:
+  imports:
+    - tpi
```

Maybe add some more imports to double check the dependencies? It doesn't seem like just `import tpi` really exercises the code (`__init__.py` doesn't seem to do too much).
```suggestion
    - tpi
    - tpi.bin
    - tpi.terraform
```

---

Was there a reason this was marked resolved without any changes?

---

I'd pushed exctly these changes but somehow lost them in the rebase, srry. Thanks for spotting it!